pkg/cli/services.go

```diff
@@ -23,6 +23,8 @@ const (
 	ServeGHCFlag string = "serve-api-ghc"
 	// ServePrimeFlag is the prime api flag
 	ServePrimeFlag string = "serve-api-prime"
+	// ServeSupportFlag is the support api flag
+	ServeSupportFlag string = "server-api-support"
```

Comment:
typo in the flag name
```suggestion
	ServeSupportFlag string = "serve-api-support"
```